### PR TITLE
gorm test logger

### PR DESCRIPTION
**Pull request by SimoneDutto**
## Description

This pr sets the gorm logger for tests.
By default they are disabled, but you can enable them via env variable or via an enable/disable methods.

Fixes _JIRA/GitHub issue number_

## Engineering checklist
*Check only items that apply*

- [ ] Documentation updated
- [ ] Covered by unit tests
- [ ] Covered by integration tests

### CODE REVIEW

File: internal/db/db.go [hunk 1]
```diff
@@ -126,6 +127,16 @@ func (d *Database) Close() error {
 	return nil
 }
 
+// EnableLogs enables gorm logging.
+func (d *Database) EnableLogs() {
```
Comment (kian99) [inline internal/db/db.go]: Should we reserve this to only tests?

File: internal/db/group_test.go [hunk 2]
```diff
@@ -53,6 +53,7 @@ func (s *dbSuite) TestAddGroup(c *qt.C) {
 	c.Assert(ge.ID, qt.Equals, uint(1))
 	c.Assert(ge.Name, qt.Equals, "test-group")
 	c.Assert(ge.UUID, qt.Equals, uuid)
+	c.Fail()
```
Comment (kian99) [inline internal/db/group_test.go]: leftover?

**Comment by ale8k** [comment]
test

File: internal/db/group_test.go (hunk 2 above)
Comment (ale8k) [inline internal/db/group_test.go]: Remove please :)

**Review by kian99 — COMMENTED** [review]
looks good with 2 comments.

File: internal/db/db.go (hunk 1 above)
Comment (SimoneDutto) [reply internal/db/db.go]: we can even remove this and use the env var when logs are needed

File: internal/db/group_test.go (hunk 2 above)
Comment (SimoneDutto) [reply internal/db/group_test.go]: y

File: internal/db/db.go (hunk 1 above)
Comment (kian99) [reply internal/db/db.go]: Happy with that

**Review by kian99 — DISMISSED** [review]
Please also add documentation on this in CONTRIBUTING.md

**Review by kian99 — APPROVED** [review]

**Review by ale8k — APPROVED** [review]
lgtm

**Pull request merged**